src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -1287,19 +1390,6 @@ private void findSig(CORINFO_MODULE_STRUCT_* module, uint sigTOK, CORINFO_CONTEX
 
             Get_CORINFO_SIG_INFO(methodSig, sig);
 
-            // CORINFO_CALLCONV_UNMANAGED is handled by Get_CORINFO_SIG_INFO
-            Debug.Assert(sig->callConv != CorInfoCallConv.CORINFO_CALLCONV_UNMANAGED);
-
-            // TODO: Replace this with a public mechanism to mark calli with SuppressGCTransition once it becomes available.
-            if (methodIL is PInvokeILStubMethodIL stubIL)
```

Comment:
This check is missing after the refactoring

Reply:
I'm not sure I fully understand what this was doing here. Is this propagating the SuppressGCTransition attribute from a P/Invoke signature to all calli's in the stub IL?

Reply:
Ok! I'll plumb things through to support this in this PR for NativeAOT.